onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlCommon.h

```diff
@@ -22,80 +22,6 @@ namespace Dml
 
     bool IsSigned(DML_TENSOR_DATA_TYPE dataType);
 
-    /** Calculates the minimum number of bytes required to store a buffer tensor with the specified type, sizes, and
```

Comment:
Nice. It can be now used via DirectMLX.h.

Reply:
Correct.